tensorflow/tools/docker/Dockerfile

```diff
@@ -29,8 +30,8 @@ RUN pip --no-cache-dir install \
 
 # Install TensorFlow CPU version.
 ENV TENSORFLOW_VERSION 0.7.1
-RUN pip --no-cache-dir install \
-    http://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-${TENSORFLOW_VERSION}-cp27-none-linux_x86_64.whl
+ADD whl/tensorflow-*.whl /tmp/
```

Comment:
The problem with doing this is that people can simply build docker in tools/docker folder anymore. They would have to know they have to run multiple hops of other scripts. Complicated.